src/include/distributed/multi_server_executor.h

```diff
@@ -104,6 +104,16 @@ typedef enum
 	CONNECT_ACTION_CLOSED = 2
 } ConnectAction;
 
+/* Enumeration that defines the different log types of cross shard queries */
+typedef enum
+{
+	MULTI_TASK_QUERY_INFO_OFF = 0,
+	MULTI_TASK_QUERY_INFO_LOG = LOG,
+	MULTI_TASK_QUERY_INFO_NOTICE = NOTICE,
+	MULTI_TASK_QUERY_INFO_WARNING = WARNING,
+	MULTI_TASK_QUERY_INFO_ERROR = ERROR
+} MultiTaskQueryLogLevels;
```

Comment:
Seems a bit superflous to copy the definitions of LOG etc here, but doesn't matter much.